Makefile

```diff
@@ -41,27 +47,37 @@ test:
 	done
 .PHONY: test
 
-prep-for-run:
+copy-k8s-sa:
+	mkdir -p ${TLS_CERTS_BASEDIR} || true
+	oc -n ${NAMESPACE} get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name} > _output/espod && \
+	oc -n ${NAMESPACE} exec -c elasticsearch $$(cat _output/espod) -- cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt > _output/ca.crt && \
+	oc -n ${NAMESPACE} serviceaccounts get-token elasticsearch > _output/sa-token && \
```

Comment:
You could also get these from the secret e.g. `oc -n ${NAMESPACE} extract secret/elasticsearch-xxxx --to=es-token-ca/` but the problem is getting the name of the secret - there are *two* sa secrets e.g. in my cluster I have both `elasticsearch-token-9hlmh` and `elasticsearch-token-pf9vl` - you have to look at the sa to determine which one is actually being used

Reply:
@jcantrill  added this code, I have just moved it to a different target.

I have changed they way how the k8s client is created. Now if the in-cluster creation fails it fall backs to kubeconfig in home dir. So no need to run this code. Although it might be useful if you want to verify the setup really works against deployed ES.